src/statd/python/yanger/yanger.py

```diff
@@ -102,7 +102,7 @@ def insert(obj, *path_and_value):
     curr[path[-1]] = value
 
 
-def run_cmd(cmd, testfile, default=None):
+def run_cmd(cmd, testfile, default=None, quiet=False):
```

Comment:
For people familiar with Python's `subprocess.run()`, I think `check` is more familiar.

```suggestion
def run_cmd(cmd, testfile, default=None, check=True):
```

Then we can change `s/subprocess.check_output(...)/subprocess.run(..., check=check).output` below.

That way, a non-zero exitcode won't raise an exception, so there's no need for the `if not quiet` guard below.

Reply:
Ah, of course, great idea thanks!